enterprise/templates/enterprise/emails/user_notification.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}{% load enterprise_string_utils %}<html>
+<body>
+<p>{% if user_name %}{% blocktrans %}Dear {{ user_name }},{% endblocktrans %}{% else %}{% blocktrans %}Hi!{% endblocktrans %}{% endif %}</p>
+<p>{% if enrolled_in.type == "program" %}
+{% blocktrans with program_url=enrolled_in.url program_name=enrolled_in.name program_branding=enrolled_in.branding|pad_right start_date=enrolled_in.start|date:"F j, Y" %}You have been enrolled in <a href="{{ program_url }}">{{ program_name }}</a>, a {{ program_branding }}program offered by {{ organization_name }}. This program begins {{ start_date }}. For more information, see <a href="{{ program_url }}">{{ program_name }}</a>.{% endblocktrans %}{% else %}
```

Comment:
For dates, it's better not to use any specific format like `F j, Y` since that is specific to English speakers. Instead, use  "DATE_FORMAT", "DATETIME_FORMAT", "SHORT_DATE_FORMAT" or "SHORT_DATETIME_FORMAT", which will pick an appropriate date format based on the active locale. ([ref](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#date))

e.g. `{{ value|date:"SHORT_DATE_FORMAT" }}`